source/frontend/src/features/properties/filter/GeographicNameInput.test.tsx

```diff
@@ -0,0 +1,170 @@
+import { act, fireEvent, render, waitFor } from '@testing-library/react';
+import { useFormikContext } from 'formik';
+import debounce from 'lodash/debounce';
+
+import { GeographicNameInput } from './GeographicNameInput';
+import { useGeographicNamesRepository } from '@/hooks/useGeographicNamesRepository';
+import TestCommonWrapper from '@/utils/TestCommonWrapper';
+import userEvent from '@testing-library/user-event';
+
+vi.mock('formik');
+vi.mock('lodash/debounce', () => ({
+  __esModule: true,
+  default: vi.fn(fn => fn),
+}));
+vi.mock('@/hooks/useGeographicNamesRepository');
+
+const mockSearchName = {
+  execute: vi.fn(),
+  error: null,
+  loading: false,
+  status: null,
+  response: null,
+};
+
+vi.mocked(useGeographicNamesRepository).mockReturnValue({
+  searchName: mockSearchName,
+});
+
+vi.mocked(useFormikContext).mockReturnValue({
+  handleBlur: vi.fn(),
+  setFieldValue: vi.fn(),
+  values: { testField: '' },
+} as unknown as ReturnType<typeof useFormikContext>);
+
+describe('GeographicNameInput', () => {
+  const setup = (props = {}) => {
+    return render(
+      <TestCommonWrapper>
+        <GeographicNameInput field="testField" placeholder="Search..." {...props} />
+      </TestCommonWrapper>,
```

Comment:
nit - I believe the common `render` function already wraps the component with TestCommonWrapper - not sure why would need to wrap it twice